### ISSUE TITLE
Recurly::Subscription  `starts_at` support

### ISSUE THREAD

**Issue by patcoll** [recurly/recurly-client-ruby#262]
Using the REST API directly I'm able to back-date a subscription with a trial period using the `starts_at` parameter. I am not able to pass this parameter to `Recurly::Subscription.create` and have it work.

Is this a simple change? Where I can I look for implementation hints on how to implement?

**Comment by bhelx** [comment]
Can you explain more how you do this with the API directly? Could you paste the urls used and xml exchanged?

**Comment by patcoll** [comment]
@bhelx For sure! Here is the success I had yesterday:

https://gist.github.com/patcoll/c6e77092cab60de7a14e17a5f9ca7b55

The following is the documentation for the REST API that contains options for `starts_at` and `trial_ends_at `: https://dev.recurly.com/docs/create-subscription

Recurly doesn't allow trials without entering payment info, so we are currently making that happen on our end. Once the user decides to pay for an account, we need to be able to back-date the trial to start and end on specific dates. The REST API call I made above successfully made a trial that ends on the date we specified.

The most important part for us is that the `trial_ends_at` ends up being correct.

Is this scenario supported by the Ruby Recurly library?
